internal/resource/config/serversettings/server_settings_resource.go

```diff
@@ -431,7 +431,10 @@ func (r *serverSettingsResource) Schema(ctx context.Context, req resource.Schema
 						Description: "If supplied, the Source ID value entered here is used for SAML 1.x, instead of being derived from the SAML 1.x Issuer/Audience.",
 						Computed:    true,
 						Optional:    true,
-						Default:     stringdefault.StaticString(""),
+						Validators: []validator.String{
+							stringvalidator.LengthBetween(40, 40),
```

Comment:
Where does this come from?

Reply:
Received an error stating this when testing the attributes - the length must be exactly 40 characters.